gdc_client/parcel/segment.py

```diff
@@ -16,7 +16,7 @@
 import sys
 
 from intervaltree import Interval, IntervalTree
-from progressbar import ProgressBar, Percentage, Bar, ETA
+from tqdm import tqdm
```

Comment:
Nice find. Do you think we should consider using this library for our other repositories like esbuild? I found it being used in data-tools a little, so it seems like something we might want in other places.

Reply:
yeah, I think it'd be a good replacement